src/libraries/Microsoft.Extensions.Hosting/src/HostingHostBuilderExtensions.cs

```diff
@@ -198,32 +199,31 @@ public static IHostBuilder ConfigureDefaults(this IHostBuilder builder, string[]
                 }
             });
 
-            builder.ConfigureAppConfiguration((hostingContext, config) =>
-            {
-                IHostEnvironment env = hostingContext.HostingEnvironment;
+                        builder.ConfigureAppConfiguration((hostingContext, config) =>
```

Comment:
This whitespace here is really off. Did something go wrong?

Reply:
Looks like when I refactored the code using VS ctrl+'.' it re-formatted this whole method and messed up indenting. I'll fix this.